tests/test_bond.py

```diff
@@ -387,11 +393,22 @@ def test_apply_csv_changes(tmp_path):
     assert og_content == mod1_content
 
     # edit the csv, add a RenameKeyGroup
+    # make sure extension files also get renamed
+
     _edit_csv(str(tmp_path / "originals_summary.csv"))
     complete_bond.apply_csv_changes(str(tmp_path / "originals_summary.csv"),
                                     str(tmp_path / "originals_files.csv"),
                                     str(tmp_path / "modified2"))
 
+    # check files df to make sure extension files also got renmaed
+    mod_files = tmp_path / "modified2_files.csv"
+    assert Path(data_root /
+        "complete/sub-01/ses-phdiff/fmap/sub-01_ses-phdiff_acq-v5_magnitude1.bval").exists() == True
+    assert Path(data_root /
+        "complete/sub-01/ses-phdiff/fmap/sub-01_ses-phdiff_acq-v5_magnitude1.bvec").exists() == True
+    assert Path(data_root /
+        "complete/sub-01/ses-phdiff/fmap/sub-01_ses-phdiff_acq-v5_magnitude1.tsv").exists() == True
```

Comment:
Are there any bids-valid tsv's available in pnc or something? The events tsvs are described here: https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html and can have a different suffix (_events.tsv)

Reply:
How do you think we should rename the .tsv if they can have different suffixes? If the user renames the suffix, then how do we want to change the .tsv file to reflect the renaming? One option is to change the .tsv stem to match the .nii stem before doing the renaming. What do you think?